collection_requirements.rst

```diff
@@ -262,6 +262,11 @@ ways:
                <https://opensource.org/licenses/BSD-2-Clause>`_ license to make it possible for
                third-party modules which are licensed incompatibly with the GPLv3 to use them.
                Please consider this use case when licensing your own ``module_utils``.
+:Code outside plugins/: Code outside `plugins/`, in particular unit tests, must be licensed with
+                        a free software license that is compatible with the
+                       `GPL-3.0-or-later <https://www.gnu.org/licenses/gpl-3.0-standalone.html>`_.
+                       Note that such code that imports GPL licensed code must be GPL licensed
+                       itself as well.
```

Comment:
This does not seem to be formatted correctly. The indentation is really wonky in the rendered version.

Reply:
Fixed in 964631699a266c6ccd3e4348b2d164a2a3a91e45.